### PR TITLE
shellcheck: add page

### PR DESCRIPTION
**Pull request by gpenghe**
----

- [X] The page (if new), does not already exist in the repo.

- [X] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [X] The page has 8 or fewer examples.

- [X] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [X] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/shellcheck.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+> Shell script static analysis tool.
+
+- Check a shell script:
+
+`shellcheck myscript.sh`
+
+- Override script's shebang (valid values: sh, bash, and ksh):
+
+`shellcheck --shell sh myscript.sh`
```
Comment (agnivade) [inline pages/common/shellcheck.md]: You can provide multiple options using |. So `{{sh|bash|ksh}}`.

File: pages/common/shellcheck.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# shellcheck
+
+> Shell script static analysis tool.
+
+- Check a shell script:
+
+`shellcheck myscript.sh`
```
Comment (agnivade) [inline pages/common/shellcheck.md]: Let us be consistent and use `file.sh` everywhere. Also please wrap it in tokens (`{{}}`)

File: pages/common/shellcheck.md (hunk 1 above)
Comment (gpenghe) [reply pages/common/shellcheck.md]: Thanks. Addressed in new commit b97b8d8.

File: pages/common/shellcheck.md (hunk 2 above)
Comment (gpenghe) [reply pages/common/shellcheck.md]: Thanks. Addressed in new commit b97b8d8.

File: pages/common/shellcheck.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+`shellcheck {{file.sh}}`
+
+- Override script's shebang:
+
+`shellcheck --shell {{sh|bash|ksh}} {{file.sh}}`
+
+- Ignore certain errors:
+
+`shellcheck --exclude SC1009 {{file.sh}}`
```
Comment (sbrl) [inline pages/common/shellcheck.md]: Perhaps `SC1009` should be enclosed in `{{` and `}}`?

File: pages/common/shellcheck.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+- Ignore multiple errors:
+
+`shellcheck --exclude SC1009,SC1073 {{file.sh}}`
```
Comment (sbrl) [inline pages/common/shellcheck.md]: Same here.

**Comment by sbrl** [comment]
Hey, @gpenghe! Thanks for the page. I've left a comment or two for you to review.

File: pages/common/shellcheck.md (hunk 4 above)
Comment (gpenghe) [reply pages/common/shellcheck.md]: Thanks. Both are fixed in 4ca9ccb.

**Pull request merged**